maven-push.gradle

```diff
@@ -35,7 +35,7 @@ def getSnapshotRepositoryUrl() {
 }
 
 project.ext.set("props", new Properties())
-if (file("push.properties").exists()) {
+if (new File("push.properties").exists()) {
```

Comment:
This change was accidental, to me switching to IntelliJ, it does not have connection to headers feature.